sdk/face/ai-vision-face-rest/review/ai-vision-face.api.md

```diff
@@ -281,9 +282,7 @@ export interface AddLargePersonGroupPersonFaceFromUrl200Response extends HttpRes
 // @public (undocumented)
 export interface AddLargePersonGroupPersonFaceFromUrlBodyParam {
     // (undocumented)
-    body?: {
-        url: string;
-    };
+    body: AddFaceFromUrlRequest;
```

Comment:
is this change from typespec?

Reply:
Yes. It's from this commit: https://github.com/Azure/azure-rest-api-specs/commit/edb7904bfead536c7aa9716d44dba15bdabd0b00#diff-82d6ba4241ec1f0a70b19adff0d902043ca4b082e606e66b1dd8484a86383711 (L121 - L161)